### PR TITLE
test-cloud-server: build and publish test-cloud-server image

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

**Review by Danielius1922 — APPROVED** [review]

**Review by Danielius1922 — APPROVED** [review]

File: .github/workflows/publishTestCloudServer.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          tags: |
+            type=schedule
+            type=ref,event=branch
+            type=ref,event=tag
+            type=ref,event=pr
+      
+      - name: Build hub-build image
+        run: make hub-build
+
+      - name: Build and push test-cloud-server docker image
```
Comment (ondrejtomcik) [inline .github/workflows/publishTestCloudServer.yml:51]: it took me 10 seconds to understand why our image is named cloud-server. Should it really be named by example application located in the different repository? what about test-bundle ?

Comment (jkralik) [reply .github/workflows/publishTestCloudServer.yml:51]: This test is independent of the bundle. It is binary which contains all tests from grpc-gateway/service. So I don't want to move it to bundle.

Comment (ondrejtomcik) [reply .github/workflows/publishTestCloudServer.yml:51]: So do you want to stick with bame cloud-server?

Comment (jkralik) [reply .github/workflows/publishTestCloudServer.yml:51]: It's a testing tool for an iotivity cloud server so yes.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**